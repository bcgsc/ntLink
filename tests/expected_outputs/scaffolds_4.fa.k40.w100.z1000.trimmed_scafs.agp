ntLink_0	1	15012	1	W	scaf3	58	15069	-
ntLink_0	15013	23435	2	W	scaf4	41	8463	+
ntLink_1	1	8913	1	W	scaf1	1	8913	+
ntLink_1	8914	39390	2	W	scaf2	47	30523	+
