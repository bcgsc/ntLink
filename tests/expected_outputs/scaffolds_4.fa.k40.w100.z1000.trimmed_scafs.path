ntLink_0	scaf3- 1N scaf4+
ntLink_1	scaf1+ 1N scaf2+
