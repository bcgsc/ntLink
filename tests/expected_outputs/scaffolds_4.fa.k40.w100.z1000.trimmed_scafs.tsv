scaf1	0	8913
scaf2	46	30523
scaf3	57	15069
scaf4	40	8463
